[2023-03-29 16:22:59,141] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:22:52.086661+00:00 [queued]>
[2023-03-29 16:22:59,154] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:22:52.086661+00:00 [queued]>
[2023-03-29 16:22:59,156] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:22:59,157] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-03-29 16:22:59,158] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 16:22:59,179] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): log_suc> on 2023-03-29T16:22:52.086661+00:00
[2023-03-29 16:22:59,185] {standard_task_runner.py:52} INFO - Started process 26675 to run task
[2023-03-29 16:22:59,201] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_22222222222', 'log_suc', '2023-03-29T16:22:52.086661+00:00', '--job-id', '331', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmp8gepj_14', '--error-file', '/tmp/tmpmbrjip8h']
[2023-03-29 16:22:59,203] {standard_task_runner.py:77} INFO - Job 331: Subtask log_suc
[2023-03-29 16:22:59,254] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_22222222222.log_suc 2023-03-29T16:22:52.086661+00:00 [running]> on host 0bc3da97cebb
[2023-03-29 16:22:59,322] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_22222222222
AIRFLOW_CTX_TASK_ID=log_suc
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T16:22:52.086661+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T16:22:52.086661+00:00
[2023-03-29 16:22:59,324] {ETL.py:53} INFO - completed
[2023-03-29 16:22:59,376] {python.py:151} INFO - Done. Returned value was: None
[2023-03-29 16:22:59,402] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=postgres_22222222222, task_id=log_suc, execution_date=20230329T162252, start_date=20230329T162259, end_date=20230329T162259
[2023-03-29 16:22:59,468] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-03-29 16:22:59,498] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
